Astro CLI 1.3.0 core doc changes (#455)
astronomer/docs: astro/deploy-code.md

astrocloud deploy
```

This command returns a list of Deployments available in your Workspace and prompts you to pick one.

> After you select a Deployment, the CLI parses your DAGs to ensure that they don't contain basic syntax and import errors. These tests are equivalent to those that run on `astro dev parse` in a local Airflow environment. If any of your DAGs fail this parse, the deploy to Astro will also fail.

Reviewer: ```suggestion
After you select a Deployment, the CLI parses your DAGs to ensure that they don't contain basic syntax and import errors. This test is equivalent to the one that runs during `astrocloud dev parse` in a local Airflow environment. If any of your DAGs fail this parse, the deploy to Astro will also fail.
```

astrocloud deploy
```

> This command returns a list of Airflow Deployments available in your Workspace and prompts you to pick one.

Reviewer 2: ```suggestion
This command returns a list of Deployments available in your Workspace and prompts you to pick one.
```

> After selecting a Deployment, the CLI runs several unit tests to ensure that your DAGs don't contain basic errors. If any of these tests fail, the deploy also fails.  
> 
> If your code passes the unit tests, the CLI builds your Astro project directory into a new Docker image pushes this image to Astro.

Reviewer 2: ```suggestion
After selecting a Deployment, the CLI runs several unit tests to ensure that your DAGs don't contain basic syntax and import errors. If any of these tests fail, the deploy will also fail. If your code passes the unit tests, the CLI builds your Astro project directory into a new Docker image pushes this image to Astro.
```

Reviewer 2: @sunkickr What's the error users will see when this stuff fails?

Reviewer: I wouldn't say it runs unit tests, Id say it parses your DAGs. the main point of the tests are to keep bad code that could take a DAG down if pushed. I wouldn't want a user to think that we are enforcing any kind of code structure with a unit tests

:::tip

> To force a deploy even if your project fails unit tests, you can run `astrocloud deploy --force`.

Reviewer: ```suggestion
To force a deploy even if your project had DAG errors, you can run `astrocloud deploy --force`.
```
